app/templates/apps/details.html

```diff
@@ -111,7 +111,7 @@ <h2 class="heading-medium">App data sources</h2>
               <form action="{{ url_for('apps3buckets.delete', {id: apps3bucket.id}) }}" method="post" class="inline-form clearfix">
                 <input type="hidden" name="redirect_to" value="{{ url_for('apps.details', {id: app.id}) }}">
 
-                <input type="submit" class="js-confirm button button-secondary right" value="Revoke access" />
+                <input type="submit" class="js-confirm button button-secondary right" value="Disconnect data source" />
```

Comment:
Is "app" implicit from the context or should this be `Disconnect app data source` (these sentences are getting a bit long...)

Reply:
In this case yeah, it should be implicit and I left it out to keep the button a bit shorter.